content/shmem_pe_quiet.tex

```diff
@@ -0,0 +1,50 @@
+\apisummary{
+    Waits for completion of all outstanding memory store, blocking 
+    \PUT{}, \ac{AMO}, and \emph{put-with-signal}, as well as 
+    nonblocking \PUT{}, \emph{put-with-signal}, and \GET{} routines 
+    to symmetric data objects issued by the calling \ac{PE} at the target
+    \acp{PE}.
+}
+
+\begin{apidefinition}
+
+\begin{Csynopsis}
+void @\FuncDecl{shmem\_pe\_quiet}@(const int *target_pes, size_t npes);
+void @\FuncDecl{shmem\_ctx\_pe\_quiet}@(shmem_ctx_t ctx, const int *target_pes, size_t
+npes);
+\end{Csynopsis}
+
+\begin{apiarguments}
+    \apiargument{IN}{ctx}{A context handle specifying the context on which to perform the operation.
+        When this argument is not provided, the operation is performed on
+        the default context.}
+	\apiargument{IN}{target\_pes}{Address of target \ac{PE} array where the
+    operations need to be completed}
+	\apiargument{IN}{npes}{The number of \acp{PE} in the target \ac{PE} array}
+
+\end{apiarguments}
+
+\apidescription{
+    The \FUNC{shmem\_pe\_quiet} ensures completion of memory store, blocking 
+    \PUT{}, \ac{AMO}, and \emph{put-with-signal}, as well as  nonblocking
+    \PUT{}, \emph{put-with-signal}, and \GET{} routines on the symmetric data
+    objects issued by the calling \ac{PE} to the target \acp{PE} and on the
+    given context.
+	
+	The completion and visibility semantics of these operations are the same as the
+	\FUNC{shmem\_quiet} routine. However, it applies only to the target
+    \acp{PE}, i.e., the routines to the target \acp{PE} are guaranteed to be
+    complete and visible to all \acp{PE} when \FUNC{shmem\_pe\_quiet} returns.
+    When calling with the \FUNC{shmem\_ctx\_pe\_quiet} interface, the updates are
+    guaranteed to be complete and visible only to the \acp{PE} in the team used to
+    create the context.
```

Comment:
WG Discussion: Consider adding an `options` flag to API; remove this restriction; and make this restriction one of the optional values.

Reply:
Will remove this (as per WG discussion) and address it via another PR. Issue is captured here : https://github.com/openshmem-org/specification/issues/487